zerver/openapi/zulip.yaml

```diff
@@ -2748,7 +2748,40 @@ paths:
                                 Event sent to a user when [message flags][message-flags] are added
                                 to messages.
 
+                                This can reflect a direct user action, or can be the indirect
+                                consequence of another action. Whatever the cause, if there's a change
+                                in the set of message flags that the user has for a message, then an
+                                `update_message_flags op: add` event will be sent with the change. Note
```

Comment:
Presumably `remove` is also possible!

Reply:
a followup: https://github.com/zulip/zulip/pull/27348#issuecomment-1777935699